content/docs/user-guide/gems/reference/environment/terrain/_index.md

```diff
@@ -29,5 +29,6 @@ To enable the Terrain Gem, do the following:
 
 | Component | Description | 
 | - | - |
+| [Terrain Physics Collider](/docs/user-guide/components/reference/terrain/terrain_physics_collider/) | Provides terrain data to a physics collider in the form of a heightfield and surface to material mapping. |
```

Comment:
```suggestion
| [Terrain Physics Collider](/docs/user-guide/components/reference/terrain/terrain_physics_collider) | Provides terrain data to a physics collider in the form of a heightfield and surface to material mapping. |
```
Links to directories should have the trailing slash, but it's not needed for pages in the `terrain` directory.  Also this link may need to be changed to use hyphens.